### PR TITLE
Repurposing content of Cockpit tour

### PR DESCRIPTION
**Pull request by MWindel**
The content of the configuration section of the old Cockpit tour has been repurposed in the existing Cockpit sections.

### CODE REVIEW

File: content/users-guide/cockpit-bundle/configuration.md [hunk 1]
```diff
@@ -48,6 +48,12 @@ You can select one of the following options for the customization of the home da
 * It is only reflected in the current custom application.
 * It is only reflected in the current user. Note that this user then needs CREATE permission for the permission type "Inventory".
 
-#### Misc
+{{< c8y-admon-info >}}
+The initial home dashboard shows a number of pre-installed widgets. These widgets can be changed according to your needs.
+{{< /c8y-admon-info >}}
+
+Click **Reset dashboard** to undo any changes to your Home dashboard. This reverts all changes to the dashboard and returns it to the initial state.
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/configuration.md:55]: Is it "home dashboard" or "Home dashboard"? I have no strong opinion here but please make it consistent.

File: content/users-guide/cockpit-bundle/dashboards.md [hunk 2]
```diff
@@ -40,7 +40,13 @@ ROLES & PERMISSIONS:
 
 ### To create a dashboard
 
-1. Select the group or the device in the navigator for which to create a dashboard.
+{{< product-c8y-iot >}} allows you to create individualized dashboards for all your devices.
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/dashboards.md:43]: I would expect this information and the one in row 45 before the actual dashboard creation process steps. 

Should be combined with this introductory sentence, perhaps preceeding it?

Dashboards provide you with a customized visualization of your data by using a set of widgets. Widgets can display maps, images, graphs, tables, and other graphic representations of data.

File: content/users-guide/cockpit-bundle/configuration.md (hunk 1 above)
Comment (MWindel) [reply content/users-guide/cockpit-bundle/configuration.md:55]:
```suggestion
Click **Reset dashboard** to undo any changes to your home dashboard. This reverts all changes to the dashboard and returns it to the initial state.
```

File: content/users-guide/cockpit-bundle/dashboards.md [hunk 3]
```diff
@@ -21,8 +21,7 @@ weight: 40
 - [Application configuration](/guides/web/#application-configuration) in the *Web SDK guide* for further information on how to customize the {{< product-c8y-iot >}} environment.
 {{< /c8y-admon-related >}}
 
-
-Dashboards provide you with a customized visualization of your data by using a set of widgets. Widgets can display maps, images, graphs, tables, and other graphic representations of data.
+{{< product-c8y-iot >}} allows you to create individualized dashboards for all your groups and devices. They provide you with a customized visualization of your data, for example, visualizing data, alarms, events and triggering remote actions, by using a set of widgets. Widgets can display maps, images, graphs, tables, and other graphic representations of data. Individual dashboards also give you the opportunity to shut off your devices remotely in case of anomalies or unwanted behavior.
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/dashboards.md:24]: They provide you with a customized visualization of your data, for example, visualizing data, alarms, events and triggering remote actions,

The second part does not really fit here: It is repetitive"customized visualization of your data, for example, visualizing data". And "triggering remote actions" is not an example for visualizing data.

Perhaps: "Dashboards provide you with a customized visualization of your data, for example, alarms and events, and allow you to trigger remote actions"

Comment (MWindel) [reply content/users-guide/cockpit-bundle/dashboards.md:24]:
```suggestion
{{< product-c8y-iot >}} allows you to create individualized dashboards for all your groups and devices. 
Dashboards provide you with a customized visualization of your data, for example, alarms and events, and allow you to trigger remote actions, by using a set of widgets. Widgets can display maps, images, graphs, tables, and other graphic representations of data.```

File: content/users-guide/cockpit-bundle/dashboards.md [hunk 4]
```diff
@@ -21,8 +21,8 @@ weight: 40
 - [Application configuration](/guides/web/#application-configuration) in the *Web SDK guide* for further information on how to customize the {{< product-c8y-iot >}} environment.
 {{< /c8y-admon-related >}}
 
-
-Dashboards provide you with a customized visualization of your data by using a set of widgets. Widgets can display maps, images, graphs, tables, and other graphic representations of data.
+{{< product-c8y-iot >}} allows you to create individualized dashboards for all your groups and devices. 
+Dashboards provide you with a customized visualization of your data, for example, alarms and events, and allow you to trigger remote actions, by using a set of widgets. Widgets can display maps, images, graphs, tables, and other graphic representations of data.```
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/dashboards.md:25]:
```suggestion
Dashboards provide you with a customized visualization of your data, for example, alarms and events, and allow you to trigger remote actions, by using a set of widgets. Widgets can display maps, images, graphs, tables, and other graphic representations of data.
```

**Review by rssag — APPROVED** [review]

**Pull request merged**